packages/rs-drive-abci/src/config.rs

```diff
@@ -8,13 +8,21 @@ pub struct PlatformConfig {
 
     /// Should we verify sum trees? Useful to set as no for tests
     pub verify_sum_trees: bool,
+
+    /// The default quorum size
+    pub quorum_size: u16,
+
+    /// How often should quorums change?
+    pub quorum_rotation_block_count: u64,
 }
 
 impl Default for PlatformConfig {
     fn default() -> Self {
         PlatformConfig {
             drive_config: Default::default(),
             verify_sum_trees: true,
+            quorum_size: 100,
+            quorum_rotation_block_count: 25,
```

Comment:
Isn't set to 15 right now?

Reply:
Doesn't really matter does it? Default is only used for tests.

Reply:
It's nothing prevents you from using it for production code. If it's only a test fixture we need to highlight it somehow and separate it from the production code.